src/main/scala/devices/i2c/I2C.scala

```diff
@@ -0,0 +1,515 @@
+// See LICENSE for license details.
+package sifive.blocks.devices.i2c
+
+import Chisel._
```

Comment:
@hcook Can you weigh in on based on our discussion just now... should we be importing Chisel or chisel3 here for new code?